client/js/InitVue.js

```diff
@@ -51,6 +50,9 @@ function initialize (components = {}, storeModules = {}, apiStoreModules = [], p
 
         const mountedEvent = new Event('vue-mounted')
         document.dispatchEvent(mountedEvent)
+        setTimeout(() => {
+          window.mounted = true
+        }, 5)
```

Comment:
Why is this necessary?

Reply:
Wouldn't it be a better solution to set the variable in an event listener, like so:

```
document.addEventListener('vue-mounted', () => {
  window.mounted = true
}
```